### ISSUE TITLE
Not all symbols are present

### ISSUE THREAD

**Issue by mleimer** [marella/material-symbols#20]
The following symbol is not part of this package:

https://fonts.google.com/icons?icon.query=progress_activity&icon.set=Material+Symbols&icon.style=Outlined

**Comment by marella** [comment]
These icons are added in the latest version 0.7.0 released today.

They were added by Google just yesterday. It usually takes about a day for the package to pick up the changes as the update script runs only once per day.
